### DISCUSSION TITLE
TailwindCSS in PHP

### DISCUSSION THREAD

**Discussion by Zaiidmoumni** [community/community#81189] [Programming Help]
### Body

Hello, 
I'm currently facing an issue with Tailwind CSS integration in my PHP project, where it seems the Tailwind styles are not building as expected. Despite following the documentation and standard procedures for setting up Tailwind with PHP, I'm encountering challenges that are impeding the successful build of styles. I've thoroughly reviewed my configuration and the order of build processes, but the issue persists. If anyone has encountered similar difficulties or has insights into troubleshooting Tailwind CSS integration within a PHP environment, I would greatly appreciate your guidance and expertise.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ananuness** [comment]
Hii! Can you tell what are you using to configure tailwind? If it's with laravel, CLI or CDN? Also, can you provide your tail config and the files you made any changes to setup it? It would help 😊

**Reply by Zaiidmoumni** [reply]
Yeah sure Here's my tail Config file , and I'm using it with just PHP no frameworks .. I've got a school project that demands that
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./Public/pages/**/*.{html,js,php}"],
  plugins: [],
  darkMode: "class",
  theme: {
    extend: {
      fontFamily: {
        poppins: ["Poppins", "sans-serif"],
      },
    },
  },
}
```
``
When I use this cdn the styles work perfectly, but it kinda disable all the JS scsript I've included in my file
```
<script src="https://cdn.tailwindcss.com"></script>
```

**Reply by ananuness** [reply]
Thank you!! First, about CDN, this is not recommended anyway (because it's not optimized), so don't worry! Now, searching more about it, you will need to do more than what's in the docs on the tailwind site!! As the vanilla php project is different from using laravel or just the steps on Tailwind CLI or PostCSS tabs in the site. 

You can check [those steps](https://www.reddit.com/r/tailwindcss/comments/184bjgz/best_approach_to_use_tailwind_css_in_a_php/) I found on reddit and see if it's missing something in your configuration. The comment in question is from **cesarcouto** he explained it better! 

If you have any questions, I'm here 😊

**Reply by ananuness** [reply]
Also, here in the content, the public folder is really "Public"? Not "public"?

>   content: ["./Public/pages/**/*.{html,js,php}"]

Just checking!

**Comment by dzenebieri** [comment]
https://tailwindcss.com/docs/installation/using-postcss
